src/dbnode/persist/fs/seek_manager.go

```diff
@@ -442,6 +442,7 @@ func (m *seekerManager) updateOpenLeaseHotSwapSeekers(
 	descriptor block.LeaseDescriptor,
 	state block.LeaseState,
 ) (*sync.WaitGroup, block.UpdateOpenLeaseResult, error) {
+	// Retrieve new seekers
```

Comment:
Super nit: Periods at end of all comments

Reply:
fixed